vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -1522,6 +1522,8 @@ class TimelineFragment @Inject constructor(
 
         observerUserTyping()
 
+        composerEditText.setUseIncognitoKeyboard(vectorPreferences.useIncognitoKeyboard())
```

Comment:
I am afraid that the `imeOptions` will be erased by the line 1530 just below, if `vectorPreferences.sendMessageWithEnter()` returns true.

Maybe on the same format than `setUseIncognitoKeyboard` create a new fun to (properly!) handle  `vectorPreferences.sendMessageWithEnter()` setting.

https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt#L1528